composer.json

```diff
@@ -6,19 +6,19 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.9",
-        "php": ">=7.2"
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9.1"
     },
     "require-dev": {
-        "composer/installers": "~1.9",
-        "cweagans/composer-patches": "^1.7",
-        "drupal/core-composer-scaffold": "^9.0",
-        "drupal/drupal-extension": "~4.1",
+        "composer/installers": "^1.11",
+        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
+        "drupal/drupal-extension": "~4.0",
```

Comment:
This was 4.1 minimum, is there a reason to lower it? Let's convert it to ^ also.